lib/chef/application/client.rb

```diff
@@ -37,6 +37,12 @@ class Chef::Application::Client < Chef::Application
     :long  => "--config CONFIG",
     :description => "The configuration file to use"
 
+  option :enable_telemetry,
+    :short => "-T",
+    :long  => "--enable-telemetry",
```

Comment:
Seems like we could make a slightly more streamlined UX here. Thinking something like this: `-T`== `-T log` == `--telemetry` == `--telemetery log`. That would allow configuring the processor(s?) via the command line too.